docs/source/get_involved/code_of_conduct.rst

```diff
@@ -342,14 +397,161 @@ Guide <https://www.djangoproject.com/conduct/>`__ and we are also
+* hiring decisions for roles that are advertised with a primary ``sktime`` branding
+* spending decisions from ``sktime`` community administered accounts
+* decisions arising from academic collaboration or commercial requests directed to the ``sktime`` community,
+  e.g., via official email, or via digital ``sktime`` discussion tools (slack, GitHub discussion etc)
+
+"resources" in the above context are to be interpreted in the usual sense of the word,
+i.e., valuable possessions, opportunities, qualities, that can be accessed and allocated.
+
+Examples include but are not limited to:
+
+* budget, money, financial credit
+* job opportunities, business opportunities
+* decision making power, soft or hard influence
+* dedicated work time of subordinates or dependents
+* material possessions of value
+* information of value
+* access to any of the above
+
+Best practice
+^^^^^^^^^^^^^
+
+To ensure decision making in line with the code of conduct, decision making must be:
+
+* by the ``sktime`` community, through ``sktime`` community decision making mechanisms.
+  E.g., following decision making outlined in the ``sktime`` governance document.
+* in communication transparent to the ``sktime`` community, following communication requirements outlined in the ``sktime`` governance document.
+* pursuant of and in line with the charitable mission of ``sktime`` and NumFOCUS.
+
+Adherence to widely accepted guidelines of anti-corruption and anti-bribery practice
+(e.g., United Nations Anti-Corruption Guidelines) is strictly expected,
+especially for resource allocation decisions of major magnitude, e.g., in hiring processes,
+or decisions in the 5-digit dollar range and upwards.
+
+This strict requirement for best practice remains unchanged even if a local context may require less,
+e.g., university administrations,
+company policies, national anti-corruption and anti-bribery laws (or their absence), etc.
+
+For instance, it is in-principle possible to misappropriate ``sktime`` resource, while
+being in formal compliance with specific national laws, institutional laws, and local policies.
+Irrespective of this, such an action would still count as a severe CoC violation.
+
+Examples of not acceptable behaviour
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* obtaining a resource opportunity through ``sktime``,
+  then retreating or resigning ``sktime`` roles to declare that the opportunity is now in a personal role and not on behalf of ``sktime``.
+* pretending to communicate on behalf of ``sktime``, or unauthorized use of the ``sktime`` brand,
+  including but not limited to violations of the BSD 3-clause license
+* soft refusal to implement best anti-corruption and anti-bribery practice, e.g.,
+  soft refusal to make budgets, invoives, or spending records transparent to the ``sktime`` community
+* attempts to suppress criticism of bad practice through invoking the code of conduct, e.g., tone policing or DARVO
```

Comment:
This too was specified above, see line 177

Reply:
I think it is so common (conditional on the violation having occurred) that it is better to repeat.

Reply:
Ok, I don't see this as a blocker, especially if it's intentional. 

However, I believe the longer the document the less likely that people are going to go through it. Our main intention here is that when people join they actually _read_ and _understand_ what the CoC implies and _agree_ to that. I think it's more likely if the CoC is written in plain and clear language: delivering clear and concise points on what's expected from the contributors. 

I do NOT claim that this is your intention, but the verboseness of CoC in my opinion contradicts the purpose a bit. That's why I'd avoid repetition and excessive elaboration. Maybe an option can be putting this in a separate document, sth like "a guide to the sktime CoCC"? That's intended for the CoCC members primarily and can be referred when making decisions/solving hard cases and contain examples, etc... Wdyt? We still should have all the points in the main CoC and only put the details, elaborations and examples in the CoCC guide.

Reply:
ok, I'm happy to splice the appendix on a separate page, if that's preferred?

Reply:
I personally would prefer that, yes.